koku/api/report/test/gcp/test_gcp_query_handler.py

```diff
@@ -1274,12 +1274,14 @@ def test_query_table(self):
                 GCPCostSummaryByServiceP,
             ),
             (
-                "?filter[service]=Virtual%20Network,VPN,DNS,Traffic%20Manager,ExpressRoute,Load%20Balancer,Application%20Gateway",  # noqa: E501
+                "?filter[service]=Virtual%20Network,VPN,DNS,Traffic%20Manager,ExpressRoute,Load%20Balancer,\
+                Application%20Gateway",
```

Comment:
This isn't quite what you want. This preserves the leading white space, so what you end up with is a string without the newline but a lot of white space in the middle:

```
'?filter[service]=Virtual%20Network,VPN,DNS,Traffic%20Manager,ExpressRoute,Load%20Balancer,                Application%20Gateway'
```

I'm surprised the test still passes 🤨.

Instead of an [escape sequence](https://docs.python.org/3/reference/lexical_analysis.html#escape-sequences), use [string literal concatenation](https://docs.python.org/3/reference/lexical_analysis.html#string-concatenation).


```suggestion
                (
                    "?filter[service]=Virtual%20Network,VPN,DNS,Traffic%20Manager,ExpressRouteLoad%20Balancer"
                    "Application%20Gateway"
                ),
```

Note the lack of `,` at the end of each line. This is a [parenthesized expression](https://docs.python.org/3/reference/expressions.html#parenthesized-forms). Adding commas between the elements would make this a tuple of strings, which is not what you want.